tests/test_hashing.py

```diff
@@ -9,7 +9,7 @@
 ALREADY_PROCESSED_PLACEHOLDER = hashing._Memoizer.ALREADY_PROCESSED_PLACEHOLDER
 
 
-@pytest.fixture(autouse=True, scope="function")
+@pytest.fixture(autouse=True)
```

Comment:
What does this change do 🤔

Reply:
Here I just removed an unneeded arg as scope="function" is the deafult.